src/Rings/MPolyQuo.jl

```diff
@@ -1215,7 +1216,7 @@ function minimal_generating_set(I::MPolyQuoIdeal{<:MPolyElem_dec}; ordering::Mon
 
   @assert isgraded(Q)
   
-  if !(base_ring(base_ring(Q)) isa AbstractAlgebra.Field)
+  if !(coefficient_ring(base_ring(Q)) isa AbstractAlgebra.Field)
```

Comment:
ahh, the joys of automatic type inference: Is `Q` here an `MPolyQuo`? If so, can't we do `coefficient_ring(Q)` here since this is implemented in this PR?

Reply:
Yup, that was a mistake on my part. Thank you for catching it!